pkgs/top-level/all-packages.nix

```diff
@@ -34212,4 +34212,7 @@ with pkgs;
   };
 
   zthrottle = callPackage ../tools/misc/zthrottle { };
+
+  pywm = callPackage ../development/python-modules/pywm {};
+  newm = callPackage ../applications/window-managers/newm {};
```

Comment:
please try to place these in a more sorted location. Appending to end just causes a lot of conflicts

Reply:
Thank you for your review; I adjusted the placing of both packages